src/Game.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './style.css';
+import Board from './Board';
+
+export const TurnContext = React.createContext();
```

Comment:
One other note - you can include multiple pieces of state on a single context, so I'd probably group both the `turn` and the `setTurn` function onto one `TurnContext` instead of separating them out.